### ISSUE TITLE
Intermittent connection issues between web and api while developing locally

### ISSUE THREAD

**Issue by hemildesai** [redwoodjs/redwood#923]
Intermittently, I get the following error in the logs of `yarn rw dev` - 
```
web | [HPM] Error occurred while trying to proxy request /graphql from localhost:8910 to http://localhost:8911 (ECONNRESET) (https://nodejs.org/api/errors.html#errors_common_system_errors)
api | POST /graphql - - ms - -
```

When it happens, the web page doesn't load anything and I see errors of the form mentioned above whenever I refresh.  I've run into this issue a few times but I haven't been able to reproduce this issue at will, so apologies for not providing the full details. Hoping that someone has had a similar experience and can provide guidance.

Let me know if there are ways to capture more information when this occurs, happy to do so and add the info here.

**Comment by hemildesai** [comment]
I looked around a bit and found https://github.com/saikat/react-apollo-starter-kit/issues/20#issuecomment-316651403

I tried it out and haven't run into this issue again. It would be nice to use this fix at https://github.com/redwoodjs/redwood/blob/main/packages/core/config/webpack.development.js#L23

**Comment by peterp** [comment]
Awesome! Thanks for finding that - would you mind contributing it?

**Comment by peterp** [comment]
Closed by #940
